lib/robots/dor_repo/release/release_members.rb

```diff
@@ -29,7 +29,7 @@ def members
         end
 
         def published?(member)
-          object_client.milestones.date(milestone_name: 'published', version: member.version).present?
+          Dor::Services::Client.object(member.externalIdentifier).milestones.date(milestone_name: 'published', version: member.version).present?
```

Comment:
I don't see what the difference is here. Is it worth a comment--especially since it seems like a pretty important difference?